compoments/report-repair/not-eligible.js

```diff
@@ -15,37 +13,19 @@ const NotEligible = () => {
       </header>
       <div className="govuk-grid-column-two-thirds">
         <h1 className='govuk-heading-xl'>{title}</h1>
-        <h3 className='govuk-heading-m'>Emergencies</h3>
-        <p>
-          An emergency is defined as something which could cause danger to
-          someone’s health or safety or cause serious damage and destruction to
-          property.
-        </p>
-        <ContactNumbers/>
-        <Details summary="Opening times" testid="opening-times">
-          <OpeningHours />
-        </Details>
-        <p>
-          Please do not call the emergency out of hours number if the repair is
-          not urgent. We may charge you a fee if the repair is not deemed an
-          emergency.
-        </p>
-        <p>
-          If you suspect you have a gas leak, you must report it immediately to
-          the Gas Emergency Service <strong>0800 111 999</strong> or via textphone (minicom)
-          on <strong>0800 371 787</strong>
-        </p>
-        <hr></hr>
-        <h3>Non-emergency requests</h3>
-        <p>
-          If you think the council is responsible for your property, please call <strong>{customerServicesTelephoneNumber}</strong>.
-        </p>
-        <p>
-          If you are a leaseholder you can still report communal repairs on <strong>{customerServicesTelephoneNumber}</strong>.
-        </p>
+        <h2 className="govuk-heading-m govuk-!-margin-bottom-2">Postcode</h2>
+        <PostcodeChange goToStep={goToStep} postcode={postcode} />
+        <h3 className="govuk-heading-m">If you think the council is responsible for this property</h3>
+        <p>Contact us via telephone {customerServicesTelephoneNumber}</p>
+        <p>Our call centre is open between {customerServicesOpeningHoursDescription}</p>
```

Comment:
Are these changes also agreed for the MVP service?

Reply:
Yes, this is going on in both mvp & full. Please see https://docs.google.com/spreadsheets/d/1jZ5M8fooTxP_lk4w-Gp7GQr5k9DD4Uvp4-O-gGI0I6M/edit?usp=sharing .